persimmon/website/urls.py

```diff
@@ -23,8 +25,20 @@
     path('api/bankaccount/new', apis.create_bank_account, name='create-bank-account'),
     path('user-lookup', apis.user_lookup, name='user-lookup'),
 
-    path('reset-password', html_views.reset_password_page),
-    path('reset-password/sent', html_views.reset_password_sent),
+    path('', include('django.contrib.auth.urls')),
+    path('password_reset',
+         auth_views.PasswordResetView.as_view(template_name="pages/reset_password.html"),
+         name="password_reset"),
+    path('password_reset/sent',
+         auth_views.PasswordResetDoneView.as_view(template_name="pages/reset_password_sent.html"),
+         name="password_reset_done"),
+    path('reset/<uidb64>/<token>',
+         auth_views.PasswordResetConfirmView.as_view(template_name="pages/reset_password_confirm.html"),
+         name="password_reset_confirm"),
+    path('reset_password/success',
+         auth_views.PasswordResetCompleteView.as_view(template_name="pages/reset_password_success.html"),
+         name="password_reset_complete"),
```

Comment:
We need to add a link to `password_reset` from the login page

Reply:
like change it to login/password _reset?